src/components/Form.js

```diff
@@ -10,13 +10,12 @@ import ErrorBoundary from 'components/Errors/ErrorBoundary';
 import FormStart from 'components/FormStart';
 import FormStep from 'components/FormStep';
 import Loader from 'components/Loader';
-import PaymentOverview from 'components/PaymentOverview';
 import ProgressIndicator from 'components/ProgressIndicator';
 import RequireSubmission from 'components/RequireSubmission';
 import {RequireSession} from 'components/Sessions';
-import SubmissionConfirmation from 'components/SubmissionConfirmation';
 import SubmissionSummary from 'components/Summary';
 import {START_FORM_QUERY_PARAM} from 'components/constants';
+import {ConfirmationView, StartPaymentView} from 'components/postCompletionViews';
```

Comment:
Can you please keep the file/folder naming consistent to use `CamelCase` naming like the rest of the codebase?

Reply:
Ahh I wasn't sure because this was a folder with 'many components' inside so I thought to give it a lower case initial like the `appointment` folder. I will change it.

Reply:
yeah after looking around it looks like we do `ComponentName` for a directory for a component (with or without private helper components) and `some-group` for more generic names (like `api-mocks` and `components`).